v2.6/getting-started/kubernetes/upgrade/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Upgrading Calico for Kubernetes
+redirect_from: latest/getting-started/kubernetes/upgrade
+---
+
+You can upgrade to Calico {{site.data.versions[page.version].first.title}} from
+Calico v2.0 or later. Upgrades from earlier versions are not supported.
+
+The steps in the upgrade vary according to the version you are upgrading from
+and your datastore type. Some current versions of Calico and datastore types
+may require you to perform a policy conversion procedure, a data migration,
+or both.  
+
+1. **[Convert policies](/{{page.version}}/getting-started/kubernetes/upgrade/convert)** 
+
+   If you are using Calico policies and you meet the following criteria, you must convert 
+   your policies first. If you are not using Calico policies, you don't need to 
```

Comment:
I think this sentence is misleading.

This step only needs to be performed if you're using the Kubernetes NetworkPolicy API.

If you're using Calico policies, but not through that API, then nothing needs to change.